inc/sainexthopgroup.h

```diff
@@ -299,6 +299,17 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_LABEL,
 
+    /**
+     * @brief Weighted multi path configuration mode.
+     * false: Nexthop group is programmed with repeated member entries proportional to their weight
+     * true: Nexthop group is programmed with switch native configuration
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
```

Comment:
The default value of "false" will NOT be backward compatible for devices that support Native WCMP.

The name of this attribute must be SAI_NEXT_HOP_GROUP_ATTR_DISABLE_NATIVE_WCMP with default value as False.